DTAConfig/OptionPanels/DisplayOptionsPanel.cs

```diff
@@ -679,20 +679,20 @@ public override void Load()
                 ddi => (string)ddi.Tag == UserINISettings.Instance.ClientTheme);
             ddClientTheme.SelectedIndex = selectedThemeIndex > -1 ? selectedThemeIndex : 0;
 
-            int selectedTranslationIndex = ddTranslation.Items.FindIndex(
-                ddi => (string)ddi.Tag == UserINISettings.Instance.Translation);
-
-            if (selectedTranslationIndex > -1)
-            {
-                ddTranslation.SelectedIndex = selectedTranslationIndex;
-            }
-            else
+            foreach (string localeCode in new string[] { UserINISettings.Instance.Translation, Translation.GetDefaultTranslationLocaleCode(), ProgramConstants.HARDCODED_LOCALE_CODE })
```

Comment:
```suggestion
            foreach (string localeCode in [UserINISettings.Instance.Translation, Translation.GetDefaultTranslationLocaleCode(), ProgramConstants.HARDCODED_LOCALE_CODE])
```

Reply:
I remember the compiler would complain about writing `[]` in a foreach loop. Not pretty sure. Will commit and wait for the bot compiling the codes

Reply:
```
Error: D:\a\xna-cncnet-client\xna-cncnet-client\DTAConfig\OptionPanels\DisplayOptionsPanel.cs(682,43): error CS9176: There is no target type for the collection expression. [D:\a\xna-cncnet-client\xna-cncnet-client\DTAConfig\DTAConfig.csproj::TargetFramework=net8.0]
```